parallel/tutorial.md

```diff
@@ -38,111 +38,121 @@ threads can be used simultaneously to do the computation, which comprises the se
 
-Using catkin requires the python `catkin-pkg` to be installed:
+```bash
+# clone gazebo9
+git clone https://github.com/osrf/gazebo -b gazebo9
+# or clone gazebo 11
+git clone https://github.com/osrf/gazebo -b gazebo11
+cd gazebo
+mkdir build
+cd build
+cmake .. -DENABLE_DIAGNOSTICS=1
+make
+sudo make install
+```
 
-~~~
-# Ubuntu
-sudo apt-get install python-catkin-pkg
-# Others
-sudo pip install catkin-pkg
-~~~
+**For a detailed version of the install instructions please visit the [install tutorial](http://gazebosim.org/tutorials?tut=install_ubuntu).**
 
-Here we create a `ws` folder as workspace under `HOME` directory. Then we clone
-catkin, gazebo and sdformat into the `ws/src` folder.
+## Run the code
 
-~~~
-export WS=${HOME}/ws/gazebo_parallel
-mkdir -p ${WS}/src
-cd ${WS}/src
-git clone https://github.com/ros/catkin.git
-git clone https://github.com/osrf/gazebo
-git clone https://github.com/osrf/sdformat
-~~~
+Threading is enabled using SDFormat parameters:
 
-Then we update gazebo and sdformat to the diagnostics related branch:
+* `island_threads`: integer number of threads to use for island threading
+* `thread_position_correction`: flag to turn threading on for ODE quickstep position error correction
 
-~~~
-cd ${WS}/src/gazebo
-git checkout diagnostics_scpeters
-cd ${WS}/src/sdformat
-git checkout island_threads
-~~~
+## Test Scenarios
 
-Next we download [package_gazebo.xml](https://bitbucket.org/scpeters/unix-stuff/raw/master/package_xml/package_gazebo.xml)
-and [package_sdformat.xml](https://bitbucket.org/scpeters/unix-stuff/raw/master/package_xml/package_sdformat.xml),
-copy them to the cloned `gazebo` and `sdformat` source folder as `package.xml`
+The effectiveness of the parallelization strategies described in this tutorial depends on the scenario that is being simulated. For example, the threaded islands strategy can parallelize simulation of multiple non-interacting robots, while the position error correction strategy can parallelize simulation of individual robot. The following test scenarios
+vary the number and complexity of simulated entities to show the performance of each parallelization strategy.
 
-~~~
-curl https://bitbucket.org/scpeters/unix-stuff/raw/master/package_xml/package_gazebo.xml > ${WS}/src/gazebo/package.xml
-curl https://bitbucket.org/scpeters/unix-stuff/raw/master/package_xml/package_sdformat.xml > ${WS}/src/sdformat/package.xml
-~~~
+* [revolute\_joint\_test.world](https://github.com/osrf/gazebo/blob/diagnostics_scpeters/test/worlds/revolute_joint_test.world#L12). The `revolute_joint_test.world` file is used in Gazebo’s automated test system. The world file includes eight instances of the double pendulum with base model arrayed in a circle. The model consists of a base in contact with the ground that is connected to two links by revolute joints. The models are arranged in close proximity but do not contact each other. This scenario includes contact, articulation constraints, and multiple *islands*.
```

Comment:
```suggestion
* [revolute\_joint\_test.world](https://github.com/osrf/gazebo/blob/diagnostics_scpeters/test/worlds/revolute_joint_test.world#L12). The `revolute_joint_test.world` file is used in Gazebo’s automated test system. The world file includes eight instances of the "double pendulum with base" model arrayed in a circle. The model consists of a base in contact with the ground that is connected to two links by revolute joints. The models are arranged in close proximity but do not contact each other. This scenario includes contact, articulation constraints, and multiple *islands*.
```

Even though I know that model, I was a bit confused reading that sentence :sweat_smile: 

Also, a general suggestion is to break all lines in 80/100 characters so it's easier to review and see diffs. The only diff above are the `""` around the model name.